# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/nsStyleStruct.h
# Commit: c1e64e8e4665
# Full Hash: c1e64e8e466506d1d9a36d747df25527bd545b10
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-12-01 16:18:29
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - Add support for `content-visibility: auto` r=emilio
#   
#   This change adds support for `content-visibilty: auto` as well as
#   showing and hiding content based on the relevancy of the content as
#   defined in the specification. Changes to relevancy are handled by
# ==============================================================================

diff -r 6d4d56f57b7b -r c1e64e8e4665 layout/style/nsStyleStruct.h
--- a/layout/style/nsStyleStruct.h	Thu Dec 01 01:23:08 2022 +0200
+++ b/layout/style/nsStyleStruct.h	Wed Nov 30 18:00:27 2022 +0000
@@ -1612,6 +1612,10 @@
     return mContentVisibility == StyleContentVisibility::Hidden;
   }
 
+  bool IsContentVisibilityAuto() const {
+    return mContentVisibility == StyleContentVisibility::Auto;
+  }
+
   /* Returns whether the element has the transform property or a related
    * property. */
   bool HasTransformStyle() const {